pkg/frontend/openshiftcluster_preflightvalidation.go

```diff
@@ -79,35 +90,45 @@ func (f *frontend) _preflightValidation(ctx context.Context, log *logrus.Entry,
 	converter := f.apis[apiVersion].OpenShiftClusterConverter
 	staticValidator := f.apis[apiVersion].OpenShiftClusterStaticValidator
 	ext := converter.ToExternal(oc)
-	if err := json.Unmarshal(raw, &ext); err != nil {
+	if err = json.Unmarshal(raw, &ext); err != nil {
 		return api.ValidationResult{
 			Status: api.ValidationStatusFailed,
-			Error: &api.ManagementErrorWithDetails{
-				Message: to.StringPtr(err.Error()),
+			Error: &api.CloudErrorBody{
+				Message: err.Error(),
```

Comment:
Same as above, if I'm reading correctly these are returned to customers - should we be sending them "Could not parse L35 C15 '}' expected ']'" errors, or is it better to log that message and return a 5xx error because our API is fundamentally broken?

Reply:
For preflight, it expects a 200 response each time. We could log the message for it though